test/files/neg/warn-unused-params.check

```diff
@@ -13,6 +13,12 @@ case class CaseyAtTheBat(k: Int)(s: String)        // warn
 warn-unused-params.scala:62: warning: parameter value readResolve in method f is never used
   def f(readResolve: Int) = 42           // warn
         ^
+warn-unused-params.scala:76: warning: parameter value i in value $anonfun is never used
+  val f = (i: Int) => 42
+            ^
+warn-unused-params.scala:78: warning: parameter value i in value $anonfun is never used
+  val g = for (i <- List(1)) yield 42
+               ^
```

Comment:
```scala
scala> val f = (_: Int) => 1
<console>:11: warning: parameter value x$1 in value $anonfun is never used
       val f = (_: Int) => 1
                 ^
```

I suggest that we shouldn't warn on unnamed (`WILDCARD`-named) params.

Reply:
Good catch.